src/components/new-request/name-check/name-check.vue

```diff
@@ -579,6 +603,9 @@ export default class NameCheck extends Vue {
     }
     return 'You can edit your name here and check it again'
   }
+  get numbersCheckUse (): Array<string> {
```

Comment:
I think you don't need this. Just use `this.getNumbersCheckUse` where it's used.